base/util.jl

```diff
@@ -448,9 +448,11 @@ MY INFO: hello world
 See also [`logging`](@ref).
 """
 function info(io::IO, msg...; prefix="INFO: ")
-    io = redirect(io, log_info_to, :info)
-    print_with_color(info_color(), io, prefix; bold = true)
-    println_with_color(info_color(), io, chomp(string(msg...)))
+    buf = IOBuffer()
+    iob = redirect(IOContext(buf, io), log_info_to, :info)
+    print_with_color(info_color(), iob, prefix; bold = true)
+    println_with_color(info_color(), iob, chomp(string(msg...)))
```

Comment:
The call to `string` here is kind of redundant now.

Reply:
Is it? I thought `chomp` needs a single string?